app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AccountPage.scala.html

```diff
@@ -46,6 +45,11 @@ <h1 class="govuk-heading-xl">@messages("account-homepage.title", undertaking.nam
             content = Text(messages("account-homepage.inset", lastDayToReport.getOrElse(sys.error("due date missing"))))
```

Comment:
can we put `if` in the content like 
content = if(isTimeToReport) Text(messages(....)))) else Text(messages(....))))